src/main/kotlin/inc/evil/d17map/controllers/ClassroomController.kt

```diff
@@ -127,7 +120,7 @@ class ClassroomController(private val classroomService: ClassroomService) {
             ApiResponse(responseCode = "400", description = "Invalid classroom data provided.")
         ]
     )
-    @GetMapping("/{buildingName}/{floorName}/{id}/photo")
+    @GetMapping("$BUILDINGS_PATH$FLOORS_PATH$CLASSROOMS_PATH/{id}/photo")
```

Comment:
Remove FLOORS_PATH (podadzą nam id classrooma, to już jednoznacznie identyfikuje piętro i upraszcza też nam całą logikę)